apps/site/assets/css/_customer-support.scss

```diff
@@ -169,3 +169,29 @@ button .waiting {
   top: -$base-spacing / 2;
   width: $base-spacing;
 }
+
+.c-expandable-block__link {
+  color: inherit;
+  text-decoration: none;
+  &:hover {
+    text-decoration: inherit;
+  }
+  &:link {
+    text-decoration: inherit;
+  }
+}
```

Comment:
You might need to add some padding or other layout tweak so the caret doesn't get pushed out :\ 

<img width="363" alt="image" src="https://user-images.githubusercontent.com/2136286/92524254-cacafd80-f1ef-11ea-9c7f-f89cfc1efabf.png">